beer-card/beer-card.js

```diff
@@ -48,10 +49,10 @@ class BeerCard extends HTMLElement {
     const config = this._config;
     const root = this.shadowRoot;
     if (hass.states[config.entity]) {
-      const list = hass.states[config.entity].attributes["list"];
+      const list = hass.states[config.entity].attributes[config.attribute] + '';
```

Comment:
If you're trying to get it as a string, I think a nicer option could be
const list = `${hass.states[config.entity].attributes[config.attribute]}`;